R4ML/R/ml.preprocess.R

```diff
@@ -258,27 +258,17 @@ r4ml.ml.preprocess <- function(
   # 6. we apply one hot encoding
   #1.
   
-  
-  is.omit.na <- !missing(omit.na)
-  
   # variable to control the next stop of ml.preprocessors
   stop_now <- FALSE
   proxy.omit.na <- function(data) {
-    r4ml_df = data
-    if (is.omit.na) {
-      r4ml_df <- as.r4ml.frame(SparkR::dropna(data, cols = omit.na), repartition = FALSE)
-      if (!r4ml_df@env$isCached) {
-        ignore <- SparkR::cache(r4ml_df)
-      }
-      cnt <- SparkR::count(r4ml_df)
```

Comment:
Is there another JIRA were this was discussed? [R4ML-226](https://ibm-jira.col.eslabs.ibm.com:8443/browse/HYDRAR-226) looks like it is a different issue.